### PR TITLE
httpie.md: add page

### PR DESCRIPTION
**Pull request by jsonbruce**
----

- [x] The page (if new), does not already exist in the repo.

- [x] The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.

- [x] The page has 8 or fewer examples.

- [x] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited

- [x] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines

### CODE REVIEW

**Comment by jsonbruce** [comment]
An awesome CLI http tool for debugging.
Strong Suggest!

File: pages/linux/httpie.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+# httpie
+
+> A human-friendly CLI HTTP tool.
+
+- GET method (default method with no request data):
```
Comment (sbrl) [inline pages/linux/httpie.md]: Perhaps `Send a GET request`.... would read better here?

File: pages/linux/httpie.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+`http {{https://api.github.com/users}}`
+
+- POST method (default method with request data):
```
Comment (sbrl) [inline pages/linux/httpie.md]: Perhaps `Send a post request with a specified body` would read better here?

File: pages/linux/httpie.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+`http {{example.org}} {{hello=World}}`
+
+- POST method with redirected input:
```
Comment (sbrl) [inline pages/linux/httpie.md]: Perhaps `Send a POST request with a body from stdin` would read better?

File: pages/linux/httpie.md [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+
+`http {{example.org}} < {{file.json}}`
+
+- PUT method:
```
Comment (sbrl) [inline pages/linux/httpie.md]: Perhaps `Send a PUT request with the specified body` would read better?

File: pages/linux/httpie.md [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+
+`http PUT {{httpbin.org/put}} {{API-Key:foo hello=world}}`
+
+- DELETE method:
```
Comment (sbrl) [inline pages/linux/httpie.md]: Perhaps `Send a DELETE request` would read better?

File: pages/linux/httpie.md [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+
+`http DELETE {{example.org/todos/7}}`
+
+- Show the whole HTTP exchange (request and response):
```
Comment (sbrl) [inline pages/linux/httpie.md]: Perhaps .....`both the request and response`.... would read better?

File: pages/linux/httpie.md [hunk 7]
```diff
@@ -0,0 +1,31 @@
...
+
+`http -v {{example.org}}`
+
+- Download file:
```
Comment (sbrl) [inline pages/linux/httpie.md]: Perhaps `Download a file` would read better here?

Comment (jsonbruce) [reply pages/linux/httpie.md]: Yes, that's a mistake

File: pages/linux/httpie.md (hunk 3 above)
Comment (jsonbruce) [reply pages/linux/httpie.md]: I think `redirected input` is different from `stdin`

File: pages/linux/httpie.md (hunk 4 above)
Comment (jsonbruce) [reply pages/linux/httpie.md]: I think just `Send a PUT request` is OK

File: pages/linux/httpie.md (hunk 3 above)
Comment (sbrl) [reply pages/linux/httpie.md]: Not that I'm aware of. `stdin` is the standard input, which is what you're redirecting when you use the `<` symbol.

File: pages/linux/httpie.md [hunk 8]
```diff
@@ -0,0 +1,31 @@
+# httpie
+
+> A human-friendly CLI HTTP tool.
```
Comment (agnivade) [inline pages/linux/httpie.md]: We usually avoid using subjective definitions. How about `A user friendly command line HTTP tool` ?

File: pages/linux/httpie.md [hunk 9]
```diff
@@ -0,0 +1,31 @@
...
+
+- Send a POST request (default method with request data):
+
+`http {{example.org}} {{hello=World}}`
+
+- Send a POST request with redirected input:
+
+`http {{example.org}} < {{file.json}}`
+
+- Send a PUT request:
```
Comment (agnivade) [inline pages/linux/httpie.md]: Let us remove the request header value here and put it in the DELETE example. And change the description to `Send a PUT request with a given json body`

File: pages/linux/httpie.md (hunk 8 above)
Comment (jsonbruce) [reply pages/linux/httpie.md]: OK. I got it.

File: pages/linux/httpie.md (hunk 9 above)
Comment (jsonbruce) [reply pages/linux/httpie.md]: I've changed descriptions. PTAL.

**Comment by sbrl** [comment]
Looks ok to me! Thanks, @jsonbruce 😺

**Pull request merged**